spec/requests/admin/orders_request_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Admin::Orders', type: :request do
+  let!(:admin) { create(:admin_user) }
```

Comment:
All of this let! aren't used in `context 'when admin not signed in' do` but you create it also for this context. It will be better move it to `context 'when admin signed in' do`